hack/bump-kubevirtci.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -e
+
+KUBEVIRTCI_TAG=$(curl -L -Ss https://storage.googleapis.com/kubevirt-prow/release/kubevirt/kubevirtci/latest)
+[[ ${#KUBEVIRTCI_TAG} != "18" ]] && echo "error getting KUBEVIRTCI_TAG" && exit 1
```

Comment:
Could you assign the magic number to a constant to make clear what is the purpose?
eg. EXPECTED_KUBEVIRTCI_TAG_LENGTH